frontend/src/editor/index.tsx

```diff
@@ -8,6 +8,7 @@ import LexicalEditor from './editor';
 
 import classes from './styles.module.css';
 import { Auth0ContextInterface } from '@auth0/auth0-react';
+import { wordEditorAPI } from '@/api/wordEditorAPI';
```

Comment:
No, this file is for the Lexical editor, which is not in the Word context. We'd need to implement this API separately for that case.